homeassistant/components/config/config_entries.py

```diff
@@ -378,6 +379,61 @@ def config_entries_progress(
     )
 
 
+@websocket_api.require_admin
+@websocket_api.websocket_command({"type": "config_entries/flow/subscribe"})
+def config_entries_flow_subscribe(
+    hass: HomeAssistant,
+    connection: websocket_api.ActiveConnection,
+    msg: dict[str, Any],
+) -> None:
+    """Subscribe to non user created flows being initiated or removed.
+
+    When initiating the subscription, the current flows are sent to the client.
+
+    Example of a non-user initiated flow is a discovered Hue hub that
+    requires user interaction to finish setup.
+    """
+
+    @callback
+    def async_on_flow_init_remove(change_type: str, flow_id: str) -> None:
+        """Forward config entry state events to websocket."""
+        if change_type == "remove":
+            connection.send_message(
+                websocket_api.event_message(
+                    msg["id"],
+                    {"type": change_type, "flow_id": flow_id},
```

Comment:
Here we send the `flow_id` parameter instead of the `flow`. Maybe that's confusing? Should we send the flow id parameter in all cases and have an additional parameter `flow` when that is available? I know that the flow id is part of the flow result.